_posts/2016-06-24-myeong-intro.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Myeong: introduction"
+date:   2016-06-24
+author: Myeong Lee
+---
+<link href="../assets/css/myeong.css" rel="stylesheet"></link>
```

Comment:
it works fine in MacDown, but I think Github page does not read a css file from MD file.. somehow..